src/Microsoft.Diagnostics.Runtime/src/Common/IAddressableTypedEntity.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Diagnostics.Runtime
+{
+    /// <summary>
+    /// Represents an addressable entity (class or struct) with associated type.
+    /// </summary>
+    public interface IAddressableTypedEntity: IEquatable<IAddressableTypedEntity>
+    {
+        ulong Address { get; }
+
+        string HexAddress { get; }
```

Comment:
Can you remove HexAddress?  It's something I shouldn't have added in the first place, and I'd like to not carry it forward.

Reply:
Yup, agree. Interface already gives Address, and HexAddress seem to be like syntax sugar to print user-friendly addresses whenever needed.